modules/dastool/scaffolds2bin/meta.yml

```
@@ -31,13 +31,13 @@ input:
   - fasta:
       type: file
       description: Fasta of list of fasta files recommended to be gathered via with .collect() of bins
-      pattern: "*.{fa,fas,fasta}"
+      pattern: "*.{fa,fa.gz,fas,fas.gz,fna,fna.gz,fasta,fasta.gz}"
   - binner:
       type: val
       description: Name of the binning software (optional)
   - extension:
       type: val
-      description: Fasta file extension (fa | fas | fasta | ...)
+      description: Fasta file extension (fa | fas | fasta | ...), without .gz suffix, if gzipped input.
```

I first thought that is a typo, but now I understand. Maybe easier:
```suggestion
      description: Fasta file extension (fa | fas | fasta | ...), but without .gz suffix, even if gzipped input.
```